package.json

```diff
@@ -52,6 +52,7 @@
     ]
   },
   "dependencies": {
+    "deep-assign": "^2.0.0",
```

Comment:
<img width="564" alt="screen shot 2017-05-21 at 9 41 19 am" src="https://cloud.githubusercontent.com/assets/6439050/26284826/c6231044-3e09-11e7-9f6c-fa96c13c48a3.png">

https://github.com/sindresorhus/deep-assign/issues/26

Not sure what the deal is here. Perhaps consider using https://www.npmjs.com/package/lodash.merge, which performs a deep merge w/o pulling in the entirety of lodash.